tasks/task104_semeval_2019_task10_closed_vocabulary_mathematical_answer_generation.json

```diff
@@ -0,0 +1,4148 @@
+{
+    "Contributors": [
+        "Amirreza Mirzaei"
+    ],
+    "Source": [
+        "semeval 2019 task10"
+    ],
+    "Categories": [
+        "Math Question Answering"
+    ],
+    "Definition": "You are given a mathematical question described with a circumscribed mathematical vocabulary. Questions in this task usually contain some variables (e.g. X, Y), equations (e.g. X+Y=3) and some relationships (X is greater than Y). Often you need to find the value of the variable using equations and relationships. you are also given 4 or 5 answer options (associated with \"A\", \"B\", \"C\", \"D\", \"E\"). Things to avoid: 1. Do not generate anything else apart from one of the following characters: 'A', 'B, 'C', 'D', 'E'. Emphasis & Caution: 1. LaTeX mathematical format is used to express equations. 2. Each question is solvable with high school math knowledge. 3. Only give one answer for each question.",
```

Comment:
> you are also 

capitalization 

>  LaTeX mathematical format is used to express equations. 

Same comment as the next task (unclear what "LaTeX mathematical format " means)..